packages/lib/src/components/Giftcard/Giftcard.tsx

```diff
@@ -74,8 +75,9 @@ export class GiftcardElement extends UIElement {
         }
     };
 
-    protected handleOrder = (order: Order) => {
-        this.elementRef._parentInstance.update({ order });
+    protected handleOrder = (response: PaymentResponse) => {
+        const order: Order = response.order;
+        this._parentInstance.update({ order });
```

Comment:
is `elementRef` not still needed?

Reply:
It's not needed since the instance is elementRef is also the element itself right?
I changed because I added type to `public elementRef: UIElement;` in UIElement, but that a type check error.

Reply:
`this.elementRef._parentInstance` & `this._parentInstance` are the same thing for the standalone Giftcard comp.
But are they the same for the Dropin?

Reply:
Update: I think they are the same

Reply:
Let me double check this. I didn't fully test in dropin and I'm now realising there might be some problems.